Scripts/run-tests.sh

```diff
@@ -57,10 +68,25 @@ set -o pipefail
 
 # build each scheme
 for SCHEME in ${SCHEMES[@]}; do
-	xcodebuild -scheme "$SCHEME" -destination "$DESTINATION" test | xcpretty
+	xcodebuild -scheme "$SCHEME" -destination "$DESTINATION" -enableCodeCoverage YES test | xcpretty
 	EXIT_CODES+=($?)
+	PROF_DIR=$(dirname $(find $BUILD_ROOT/.. -name Coverage.profdata))
+	cp $PROF_DIR/*.profraw $COVERAGE_DIR
```

Comment:
I assume these two lines are correct. Can come back and double-check if necessary.